packages/rke2-whereabouts/charts/values.yaml

```
@@ -46,5 +46,9 @@ tolerations:
 
 affinity: {}
 
+cni_conf:
```

does it have to be cni_conf? The convention is to use camelCase, not underscores. Maybe just `cni` or `cniConf`?
```suggestion
cni:
```

---

good point! I just copied what Multus is using